src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderNewTemplate.java

```diff
@@ -57,6 +57,7 @@
     public final String bindPorts;
     public final boolean bindAllPorts;
     public final boolean privileged;
+    public final boolean togglemode;
```

Comment:
Why not call var 'inclusive' or 'exclusive' (depends on what better fit to code)?